src/libs/invoke/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+file (GLOB SOURCES *.c)
+add_lib (invoke SOURCES ${SOURCES})
```

Comment:
As far as I can tell, we also need to link `elektra` here, since this lib contains the code for `elektraPluginOpen` and `elektraPluginClose`:

```cmake
add_lib (invoke SOURCES ${SOURCES} LINK_ELEKTRA elektra)
```

.